packages/wallet-adapter-react/src/WalletProvider.tsx

```diff
@@ -27,13 +27,14 @@ import type {
   AvailableWallets,
 } from "@aptos-labs/wallet-adapter-core";
 import { WalletCore } from "@aptos-labs/wallet-adapter-core";
+import { DappConfig } from "@aptos-labs/wallet-adapter-core/dist/WalletCore";
```

Comment:
```suggestion
import { DappConfig } from "@aptos-labs/wallet-adapter-core";
```

Reply:
I think youd need to export the DappConfig type from the core package to use it.